tests/e2e/api/fungibleAsset.test.ts

```diff
@@ -26,6 +26,11 @@ describe("FungibleAsset", () => {
     expect(data[0].asset_type).toEqual(APTOS_COIN);
   });
 
+  test("it should fetch a specific fungible asset metadata", async () => {
```

Comment:
so not sure how useful this test is, we create only one asset. maybe worth trying to get by an asset that doesnt exist and then make sure it returns nothing

Reply:
Good call!